solr/test-framework/src/java/org/apache/solr/cloud/AbstractBasicDistributedZk2TestBase.java

```diff
@@ -56,6 +58,10 @@ public abstract class AbstractBasicDistributedZk2TestBase extends AbstractFullDi
   private static final String ONE_NODE_COLLECTION = "onenodecollection";
   private final boolean onlyLeaderIndexes = random().nextBoolean();
 
+  @ClassRule
+  public static TestRule syspropRestore =
+      new TestRuleRestoreSystemProperties(NUMERIC_DOCVALUES_SYSPROP, PRS_DEFAULT_PROP);
```

Comment:
AFAIK, this shouldn't be necessary because SolrTestCase (root of all Solr tests) already declares SystemPropertiesRestoreRule as a ClassRule and also STCJ4 has a test rule for this as well for test/method level restoration.  Even LuceneTestCase adds a ClassRule for TestRuleRestoreSystemProperties, thus STC's is maybe redundant.

Reply:
Ahhh that makes sense. I'll remove the custom ones.